iree/tools/BUILD

```diff
@@ -67,10 +67,46 @@ cc_binary(
     ],
 )
 
+cc_library(
+    name = "PassesAndDialects",
+    hdrs = [
+        "init_passes.h",
+        "init_dialects.h",
+    ],
+    deps = [
+        "@llvm-project//mlir:AffineOps",
+        "@llvm-project//mlir:Analysis",
+        "@llvm-project//mlir:FxpMathOps",
+        "@llvm-project//mlir:GPUDialect",
+        "@llvm-project//mlir:GPUToSPIRVTransforms",
+        "@llvm-project//mlir:GPUToVulkanTransforms",
```

Comment:
I think this is not needed.

Reply:
Dropped the dep in Bazel as well as in CMake.